openml-python-common/src/main/java/com/feedzai/openml/python/modules/SharedModulesParser.java

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright (c) 2018 Feedzai
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
```

Comment:
Please make sure this copyright is the same as in other files in this repo

Reply:
?
it's the same

Reply:
It was a request to make sure that was the case :)